### ISSUE TITLE
Excluded roles are not working for me

### ISSUE THREAD

**Issue by stpaultim** [backdrop-contrib/stickynav#7]
Hey, it's me again. :-)

Thanks. Previous problems are fixed and stickynav is now pretty useful for me. But, I just tested the excluded roles feature (which is not really critical for me) and it seems to be broken. 

I selected "editor" and "administrator" as excluded roles, but the nav is still sticky. I'm applying it to a contrib theme, Tatsu - which is a sub-theme of Basis. 

I checked and the config is being registered in my config files, but haven't gotten any further in debugging this. 

![Sticky_Nav_Settings___localhost](https://user-images.githubusercontent.com/3144571/66250203-0ff3cf00-e705-11e9-8cea-c707d8219191.png)

**Comment by stpaultim** [comment]
Following error messages:

```
LOCATION:  http://localhost:32788/admin/config/user-interface/stickynav
REFFERER: http://localhost:32788/admin/modules?search=stick
MESSAGE:     Warning: Invalid argument supplied for foreach() in form_type_checkboxes_value() (line 2434 of /app/src/core/includes/form.inc).
```

```
LOCATION | http://localhost:32788/
REFFER | http://localhost:32788/admin/config/user-interface/stickynav
MESSAGE | Notice: Array to string conversion in stickynav_init() (line 16 of /app/src/modules/contrib/stickynav/stickynav.module)
```

**Comment by laryn** [comment]
I'll admit I've never used that feature. Can you test this:
https://github.com/backdrop-contrib/stickynav/commit/2e29c02c1c313c9432e35e747abb5ac60c0123a8

**Comment by stpaultim** [comment]
Works for me.

The reason I tried the exclude roles is that sticky nav does not work with "Keep bar at top of page" for admin menu. If you have a sticky admin menu and then enable sticky nav, stickynav covers the admin menu once you start to scroll. 

So, I checked to see if I could turn off stickynav for admins. 

This is not a critical issue for me, I could have worked with it either way. I am not sure of any other use case for excluding roles for a sticky nav. 

I will create an issue about stickynav and admin menu, but I think it's a bit of a fringe case and a solution is NOT urgent or important for me, especially because exclude roles is working.
